common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -58,7 +58,7 @@ public class CommonValue {
         public static final String ML_MODEL_GROUP_INDEX = ".plugins-ml-model-group";
         public static final String ML_MODEL_INDEX = ".plugins-ml-model";
         public static final String ML_TASK_INDEX = ".plugins-ml-task";
-        public static final Integer ML_MODEL_GROUP_INDEX_SCHEMA_VERSION = 2;
+        public static final Integer ML_MODEL_GROUP_INDEX_SCHEMA_VERSION = 3;
```

Comment:
Note to self as a reminder.  When we eventually get this on main we'll have to make sure there aren't any other conflicting schema updates.

Reply:
+1